src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -370,7 +370,7 @@ object Boot extends IOApp with LazyLogging {
     val tosService = new TosService(directoryDAO, config.termsOfServiceConfig)
     val userService = new UserService(directoryDAO, cloudExtensionsInitializer.cloudExtensions, config.blockedEmailDomains, tosService)
     val statusService =
-      new StatusService(directoryDAO, cloudExtensionsInitializer.cloudExtensions, DbReference(config.samDatabaseConfig.samRead.dbName, implicitly), 10 seconds)
+      new StatusService(directoryDAO, cloudExtensionsInitializer.cloudExtensions, 10 seconds, 1 minute)
```

Comment:
Do we maybe want to make this something configurable via env vars? just a thought

also i love 'implicitly' as an argument 😆

Reply:
Not a bad idea, but I'm not going to bother for now.  Maybe this will bight us later, but I also don't see it needing to change regularly or vary between environments or anything like that, so I think having it static for now is OK